src/containers/Dashboard.vue

```diff
@@ -57,18 +57,18 @@
       @skip="welcomeHidden = false"
       v-if="welcomeHidden"
     >
-    </tourwindow>
-    <tourwindow
+    </tour-window>
+    <tour-window
       :title="congrats"
       :text="congratsBodytext"
       :exit="congratsExit"
       :buttonText="congratsButtonText"
       @hide="showTourEndWindow = false"
       v-if="showTourEndWindow"
     >
-    </tourwindow>
+    </tour-window>
     <div>
-      <bottombar
+      <bottom-bar
```

Comment:
Without declaring the name of the component as in the old style (`Vue.component('bottombar', BottomBar);`), how do we know that bottom-bar refers to the component BottomBar? Just curious!

Reply:
It's some standard case transform vue performs. CamelCase to lower-case-with-dash